openrtb_ext/imp_appnexus.go

```diff
@@ -1,20 +1,25 @@
 package openrtb_ext
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/prebid/prebid-server/util/typeutil"
+)
 
 // ExtImpAppnexus defines the contract for bidrequest.imp[i].ext.prebid.bidder.appnexus
 type ExtImpAppnexus struct {
-	LegacyPlacementId       int                     `json:"placementId"`
+	DeprecatedPlacementId   typeutil.StringInt      `json:"placementId"`
 	LegacyInvCode           string                  `json:"invCode"`
 	LegacyTrafficSourceCode string                  `json:"trafficSourceCode"`
-	PlacementId             int                     `json:"placement_id"`
+	PlacementId             typeutil.StringInt      `json:"placement_id"`
 	InvCode                 string                  `json:"inv_code"`
 	Member                  string                  `json:"member"`
 	Keywords                []*ExtImpAppnexusKeyVal `json:"keywords"`
 	TrafficSourceCode       string                  `json:"traffic_source_code"`
 	Reserve                 float64                 `json:"reserve"`
 	Position                string                  `json:"position"`
 	UsePmtRule              *bool                   `json:"use_pmt_rule"`
+	UsePaymentRule          *bool                   `json:"usePaymentRule"`
```

Comment:
This parameter has no usages, do we really need to add it?

Reply:
> 991604b

@Sonali-More-Xandr this is done here - https://github.com/prebid/prebid-server/pull/2617/commits/991604bcf26e15a5a80445470a37033b22b40beb

Reply:
@VeronikaSolovei9 thank you for your review. We need usePaymentRule here to be in consistent with PBJS input params.

Reply:
Ok, please don't forget to add it to appnexus.json for consistency